release-latest-versions.sh

```diff
@@ -35,7 +35,7 @@ CORE_JSON="$(wget -q -O- "https://packagist.org/packages/johnpbloch/wordpress-co
 
 # @TODO Use branches!
 for MINOR in                         4.7 4.8 4.9 \
-         5.0 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9; do
+         5.0 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9 6.0; do
```

Comment:
Thank you for your PR.
I'm sorry to say that this is automatic repo (manually automatic).
But give this PR a try.

Please add 6.0 in a new line (like 5.0)